client/src/components/Users/Auth/SetNewPassword.tsx

```diff
@@ -10,53 +10,135 @@ import PasswordInput from './PasswordInput';
 import { UserContext } from '../../../providers';
 import { APP_API_BASE_URL } from '../../../configs';
 
+import SetNewPasswordImg from '../../../assets/set-new-password.svg';
+
 const useStyles = makeStyles((theme: Theme) => {
-  const xPadding = 12;
+  // const xPadding = 12;
```

Comment:
We can safely remove this, I think
```suggestion

```

Reply:
Sounds good.  These commented out styles are removed now.